src/Pages/Overview/Overview.js

```diff
@@ -213,7 +214,25 @@ export default function Overview(props) {
       }
       } />
       <div className='px-4'>
-        <div className='px-6 mt-8 border rounded-lg border-white/10'>
+        <button className="my-8 btn btn-primary lg:max-w-[20%]" onClick={async () => {
+          const result = await getAllUsersValidVerifiedAndSubscribed(props.user.token);
+          if (result.error) {
+            return alert(
+              'unable to download email list.' + result.error
+            );
+          }
+          const data = ['email', ...result.responseData.emails].join('\n');
+          const blob = new Blob([data], { type: 'text/csv' });
+          const date = new Date().toISOString().split('T')[0].replace(/-/g, '');
```

Comment:
instead of having the frontend convert the reponse to csv, how about the api just returns csv data as is?

https://gist.github.com/nulltask/2056783